Task Names with forward slash break

I have a task called "Read/Write Task"

When I select "Edit Task" in the treeview, I get the followig error:

```
notificationsAlerts.ts:42 Error running command influxdb.editTask: A system error occurred (ENOENT: no such file or directory, open '/var/folders/yb/bkwm49zj2jg4lx_m_7j12br80000gn/T/ddd2398fb4a7cb8a162e/Read/Write Task.flux'). This is likely caused by the extension that contributes influxdb.editTask.
```

The task does not open.

--- a/src/controllers/AddScriptController.ts
+++ b/src/controllers/AddScriptController.ts
@@ -83,8 +83,7 @@ export class AddScriptController {
         }
         const tmpdir = path.join(os.tmpdir(), crypto.randomBytes(10).toString('hex'))
         await fs.mkdir(tmpdir)
-        const fileExtension = 'flux'
-        const newFile = vscode.Uri.parse(path.join(tmpdir, `${script.name}.${fileExtension}`))
+        const newFile = vscode.Uri.parse(path.join(tmpdir, `${script.name.replace(path.sep, '-')}.flux`))
         await fs.writeFile(newFile.path, '')
         const document = await vscode.workspace.openTextDocument(newFile.path)
         const self = this // eslint-disable-line @typescript-eslint/no-this-alias
@@ -142,8 +141,7 @@ export class AddScriptController {
 
         const tmpdir = path.join(os.tmpdir(), crypto.randomBytes(10).toString('hex'))
         await fs.mkdir(tmpdir)
-        const fileExtension = 'flux'
-        const newFile = vscode.Uri.parse(path.join(tmpdir, `${message.name}.${fileExtension}`))
+        const newFile = vscode.Uri.parse(path.join(tmpdir, `${message.name.replace(path.sep, '-')}.flux`))
         await fs.writeFile(newFile.path, '')
         const document = await vscode.workspace.openTextDocument(newFile.path)
         const self = this // eslint-disable-line @typescript-eslint/no-this-alias

--- a/src/views/TreeView.ts
+++ b/src/views/TreeView.ts
@@ -246,7 +246,7 @@ export class Task extends vscode.TreeItem {
         // file, and then remove the entire dir with fs.rmdir.
         const tmpdir = path.join(os.tmpdir(), crypto.randomBytes(10).toString('hex'))
         await fs.mkdir(tmpdir)
-        const newFile = vscode.Uri.parse(path.join(tmpdir, `${this.task.name}.flux`))
+        const newFile = vscode.Uri.parse(path.join(tmpdir, `${this.task.name.replace(path.sep, '-')}.flux`))
         await fs.writeFile(newFile.path, '')
         const document = await vscode.workspace.openTextDocument(newFile.path)
         const self = this // eslint-disable-line @typescript-eslint/no-this-alias
@@ -353,7 +353,7 @@ export class Tasks extends vscode.TreeItem {
         // file, and then remove the entire dir with fs.rmdir.
         const tmpdir = path.join(os.tmpdir(), crypto.randomBytes(10).toString('hex'))
         await fs.mkdir(tmpdir)
-        const newFile = vscode.Uri.parse(path.join(tmpdir, `${name}.flux`))
+        const newFile = vscode.Uri.parse(path.join(tmpdir, `${name.replace(path.sep, '-')}.flux`))
         await fs.writeFile(newFile.path, '')
         const document = await vscode.workspace.openTextDocument(newFile.path)
         const self = this // eslint-disable-line @typescript-eslint/no-this-alias

